src/Sql/Sql.sqlproj

```diff
@@ -454,9 +417,46 @@
     <Build Include="SecretsManager\dbo\Tables\ServiceAccount.sql" />
     <Build Include="SecretsManager\dbo\Views\ApiKeyDetailsView.sql" />
     <Build Include="SecretsManager\dbo\Views\ApiKeyView.sql" />
-    <Build Include="dbo\Stored Procedures\Organization_ReadByLicenseKey.sql" />
-    <Build Include="dbo\Stored Procedures\Organization_ReadSelfHostedDetailsById.sql" />
-    <Build Include="dbo\Stored Procedures\OrganizationUser_ReadOccupiedSeatCountByOrganizationId.sql" />
+    <Build Include="Vault\dbo\Functions\CipherDetails.sql" />
```

Comment:
@kspearrin added the folders to `Folder` include in https://github.com/bitwarden/server/pull/2627/files. Do we want to add add the Vault directories to the sqlproj?

Reply:
Ah, yeah I'll do that as well!